jdbc/pom.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.qubole</groupId>
+    <artifactId>jdbc-jar-test</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <repositories>
+        <repository>
+            <id>s3-maven-qubole-releases</id>
+            <name>Qubole Release Repository on S3</name>
+            <url>http://maven-qubole.s3-website-us-east-1.amazonaws.com/maven/release</url>
+        </repository>
+        <repository>
+            <id>s3-maven-qubole-snapshots</id>
+            <name>Qubole Snapshot Repository on S3</name>
+            <url>http://maven-qubole.s3-website-us-east-1.amazonaws.com/maven/snapshot</url>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <build>
```

Comment:
Is this section required ? Typically it is required if you release to S3. Since this project is reading, it should not be required.

Reply:
right. It is not required. Will remove it.